recipes/autometa/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "Autometa" %}
+{% set sha256 = "b9b82c15e6f4d76c49d3e80d38a901606e7fd8ba945f83657535d27bb3d8860b" %}
+{% set version = "2.0.0a0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/KwanLab/Autometa/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --ignore-installed --no-deps -vv
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - setuptools
+  run:
+    - python >=3.7
+    - biopython
+    - bedtools
+    - bowtie2
+    - diamond
+    - hmmer
+    - hdbscan
+    - openssl ==1.0.*
```

Comment:
Is there a compelling reason for why you need to pin to this particular version rather than using the version we pin to in every other package?

Reply:
This is pinned for `samtools` to run properly